lms/djangoapps/ccx/views.py

```diff
@@ -459,40 +459,26 @@ def ccx_schedule(request, course, ccx=None):  # pylint: disable=unused-argument
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @coach_dashboard
-def ccx_invite(request, course, ccx=None):
+def ccx_students_management(request, course, ccx=None):
     """
-    Invite users to new ccx
+    Manage the enrollment of the students in a CCX
     """
     if not ccx:
         raise Http404
 
-    action = request.POST.get('enrollment-button')
-    identifiers_raw = request.POST.get('student-ids')
-    identifiers = _split_input_list(identifiers_raw)
-    email_students = 'email-students' in request.POST
-    course_key = CCXLocator.from_course_locator(course.id, unicode(ccx.id))
-    email_params = get_email_params(course, auto_enroll=True, course_key=course_key, display_name=ccx.display_name)
-
-    ccx_students_enrolling_center(action, identifiers, email_students, course_key, email_params, ccx.coach)
-
-    url = reverse('ccx_coach_dashboard', kwargs={'course_id': course_key})
-    return redirect(url)
-
+    list_action = request.POST.get('student-action', None)
```

Comment:
Nit: I think by refactoring this a bit, it could be clearer what is happening, e.g. separating out the if/else into separate functions.

Also, why is does `student-action` become `list_action`?